src/components/Loader/Navbar.tsx

```diff
@@ -0,0 +1,30 @@
+import { NavLink } from 'react-router-dom';
+import React from 'react';
+import classNames from 'classnames';
+
+const activeLinkClasses = ({ isActive }: { isActive: React.ReactNode }) => {
+  return classNames('navbar-item', { 'has-background-grey-lighter': isActive });
+};
+
+export const Navbar = () => {
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink className={activeLinkClasses} to="/">
```

Comment:
You're calling `activeLinkClasses` without any arguments. However, this function expects an object with a property `isActive`. Please ensure you're using this function correctly.